examples/plot_kernel_approximation.py

```diff
@@ -57,6 +57,80 @@
 from sklearn.kernel_approximation import (RBFSampler,
                                           Nystroem)
 from sklearn.decomposition import PCA
+from collections import defaultdict
+
+
+def timing(callable, *args, **kwargs):
+    """Time the call of a function and return time and
```

Comment:
Please adhere to PEP257: a brief description of the function should occupy only the first line of the docstring.